src/ui/components/trinary_input_string.h

```diff
@@ -54,4 +54,11 @@ component_t* trinary_input_string_create_password(
     void (*cancel_cb)(void* param),
     void* cancel_callback_param);
 
+/**
+ * Only applicable in wordlist-mode.
+ * Sets the current word. The user can then accept it or delete characters.
+ * The word must be in the wordlistm, otherwise this function aborts.
```

Comment:
"wordlist" typo. also, consider being more explicit: "must be in the component's data->wordlist" for people like me.

Reply:
done